.github/workflows/slither.yml

```diff
@@ -0,0 +1,45 @@
+name: Slither Analysis
+
+on:
+  pull_request:
+    branches: [main, release/**]
+  push:
+    branches: [main, release/**]
+    tags: [v*]
+
+permissions:
+  contents: read
+  security-events: write
+
+jobs:
+  analyze:
+    name: Slither Static Analysis
+    runs-on: smart-contracts-linux-large
+    
+    steps:
+      - name: Harden Runner
+        uses: step-security/harden-runner@cb605e52c26070c328afc4562f0b4ada7618a84e # v2.10.4
+        with:
+          egress-policy: audit
+
+      - name: Checkout repository
+        uses: actions/checkout@11bd71901bbe5b1630ceea73d27597364c9af683 # v4.2.2
+        with:
+          submodules: recursive
+
+      - name: Run Slither
+        uses: crytic/slither-action@f197989dea5b53e986d0f88c60a034ddd77ec9a8 # v0.4.0
+        id: slither
+        with:
+          node-version: 20
+          sarif: results.sarif
+          fail-on: high
+          slither-args: "--include-paths system-contracts --exclude-dependencies --exclude-informational --exclude-optimization --exclude-low --exclude-medium"
```

Comment:
Just for reference, should we include also info, low and medium?

We already `fail-on: high`, so this won't affect the result.